### DISCUSSION TITLE
Feature request: wildcard exclusions

### DISCUSSION THREAD

**Discussion by cerebrate** [jseidl/magic-areas#178] [Ideas / Feature Requests]
...of entities from areas, and possibly doing this globally rather than per-area.

I've been using thermal comfort - https://github.com/dolezsa/thermal_comfort - which creates various temperature-related entities, and since I have local temperature and humidity sensors, those get created in every area. It's not a problem to exclude all the *_thermal_comfort_* entities manually, but it would be nice if there was a central spot where I could tell Magic Areas to ignore that pattern everywhere, or indeed and perhaps more simply, ignore all entities created by integration thermal_comfort.

(If this is implemented, it might be useful to include a default blacklist of integrations/patterns with Magic Areas; these integrations tend to produce computed values which you often want to be based on the averages MA itself provides, which means that including them puts you in a situation where your sensors look broken because the integrations are caught in an I-depend-on-you-depend-on-me loop.)

**Comment by cerebrate** [comment]
Just to reinforce this a bit, this is what the exclusion list for my office currently looks like:

![image](https://user-images.githubusercontent.com/371623/229371867-c268abb1-44ba-4be5-9350-9ae463df59ef.png)
![image](https://user-images.githubusercontent.com/371623/229371875-22dd1933-2d29-4aee-8bbc-a12dc3bf9f97.png)

It would be really useful to be able to exclude on a wildcard or per-integration basis, because as you can see, having to exclude all the computed or specialist values produced by certain integrations doesn't take long to go from the sublime to the ridiculous...

**Comment by jseidl** [comment]
For sure, I 100% get it. It should be doable, already upvoted!

**Reply by cerebrate** [reply]
Thanks. Sorry if that came off as a mite naggy; just thought an illustration of scale might be useful.
